### DISCUSSION TITLE
Should we have a badass logo?

### DISCUSSION THREAD

**Discussion by paddyroddy** [UCL-ARC/python-tooling#144] [General]
Consider asking @dstansby whether his partner would be up for this challenge 👀

**Comment by samcunliffe** [comment]
Source material from the lost discussion....

![arc-logo-python-colours](https://github.com/UCL-ARC/python-tooling/assets/1836192/6c53354b-ff87-4d40-8f40-1b9ee9ce6b61)
![Python-logo-notext](https://github.com/UCL-ARC/python-tooling/assets/1836192/3db25288-3f66-4a22-bbf7-4bbd36b0298e)
![ucl-logo](https://github.com/UCL-ARC/python-tooling/assets/1836192/fa851938-5e19-411a-bbc6-04b9ee6917e4)

**Comment by paddyroddy** [comment]
I'm picturing a python wrapped around ARC, eating a cookie?

**Comment by mxochicale** [comment]
Just sketched this one? Any feedback?  
![logo-idea-v00](https://github.com/UCL-ARC/python-tooling/assets/11370681/5344cf30-ab4c-4c6e-bc7d-89a695bf98c5)

**Comment by paddyroddy** [comment]
Let's discuss in Slack

**Comment by mxochicale** [comment]
I think discussions is a suitable place to find some agreement. Just created this one https://github.com/UCL-ARC/python-tooling/discussions/133

**Comment by paddyroddy** [comment]
FYI you can convert an issue to a discussion
<img width="212" alt="image" src="https://github.com/UCL-ARC/python-tooling/assets/15052188/4f4658b0-6196-46bb-a4e6-cc2ac3e79269">

**Comment by mxochicale** [comment]
The following is a screenshot of the vector file[1] showing some of the initial ideas designed in Inkscape. I guess there is no strict plan but to propose, to iterate, to refine and to converge of what we consider suitable. 

![few-options-to-brainstorm-3-aug-2023-0848](https://github.com/UCL-ARC/python-tooling/assets/11370681/20c93c5a-f488-44d8-8357-56ea32302bb8)
[1]The size of the vector file, svg, is currently 9.5MB which I happy to put it anywhere.

**Reply by paddyroddy** [reply]
I vote 1

**Reply by dstansby** [reply]
I vote 3

**Reply by alessandrofelder** [reply]
I vote 2 (but I'd like someone to incorporate ucl and arc into that :D)

**Reply by paddyroddy** [reply]
Just to check @alessandrofelder, when you say `2` do you mean `1`? For some reason, these are zero-indexed 😆

**Reply by alessandrofelder** [reply]
Yes I mean 1

**Reply by samcunliffe** [reply]
Could we take the portico element and add it to 1 somehow? 
Even without the UCL flavour, I'm also fan of 1.

**Reply by matt-graham** [reply]
I think I did play around a bit with adding the portico, will dig out the Inkscape file. One thing that put me off using is that the terms of reuse of UCL logo are stricter than for Python logo which has more permissive terms
> In general, we want the logo to be used as widely as possible to indicate use of Python or suitability for Python. However, please [ask first](mailto:psf-trademarks@python.org) when using a derived version of the logo or when in doubt. _(https://www.python.org/community/logos/)_

**Reply by matt-graham** [reply]
![image](https://github.com/UCL-ARC/python-tooling/assets/6746980/bc56c171-8ca2-4592-bb49-6e05fd952d5b)

Some ideas on similar line to 1 above incorporating portico element. One issue is that it's difficult to get the portico to look good at small scale when combined with Python logo. Also maybe we explicitly want to _not_ include a UCL element given I assume we would be open to non-UCL contributions?

**Comment by samcunliffe** [comment]
Nice!

Resolved via #277  and #279
